graphics/lvgl/Makefile

```diff
@@ -105,13 +107,14 @@ $(LVGL_UNPACKNAME): $(LVGL_TARBALL)
 	$(Q) mv	lvgl-$(LVGL_VERSION) $(LVGL_UNPACKNAME)
 	$(Q) touch $(LVGL_UNPACKNAME)
 
-# Download and unpack tarball if no git repo found
+#Download and unpack tarball if no git repo found
```

Comment:
I think other places have space after `#`

Reply:
Hi @pkarashchenko , I think that is resolved in the latest commit, and also the logic is changed to check if "lvgl/lvgl.h" is present to skip the download of tar file.